database.rules.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    ".read": "auth != null",
```

Comment:
This is OK for now. You'll want to further limit this later so that users can only read / write their own data.